### DISCUSSION TITLE
How to change the color of the backend ?

### DISCUSSION THREAD

**Discussion by Sparfel** [nasirkhan/laravel-starter#256] [Q&A]
Hi,
I'm trying to change the color of the backoffice. So, I modify some scss files and  if i run the command
**npm run dev**
It's break the website, we loose the CSS and the JS files.
Only fews script are generated ...
We loose some css files, or the css files is shorter and we loose the icons.
Thanks for your help.

**Comment by nasirkhan** [comment]
First you should check the [webpack.mix.js](https://github.com/nasirkhan/laravel-starter/blob/master/webpack.mix.js) to understand how the backend.css is being generated. Then try to build as it is, if the build is successful, then try to change and build again. 

To modify the theme you should add changes to https://github.com/nasirkhan/laravel-starter/blob/master/resources/sass/backend.scss file. Which then build to CSS and then icon css files being added with that.

**Reply by Sparfel** [reply]
But without any change, if you do
npm run dev 
and then Schift F5 (to force to load the css again and not using the cache)
The generated css files are not like the original

**Comment by nasirkhan** [comment]
If you run `npm run dev` just after installing the application, it should throw errors.  

First you have to run `npm install` and then  `npm run dev`. 

I prefer `yarn` commands. So I install the application and run the following commands to to rebuild the frontend and backend assets. 

Following command installs the required npm packages

```
yarn
```

When the download and install of the packages is done, then run the following command, 

```
yarn dev
```

**Reply by Sparfel** [reply]
Hi,
Thanks. It works.
I had  made changes in webpack.mix.js and I had broken the site. I replaced them with original files and i solved my problem.
Now, it's OK.
Then I modify the backend style with the file **backend-custom.scss** and yarn dev compile it. 
Many thanks.
